src/azul/plugins/repository/tdr_anvil/__init__.py

```diff
@@ -60,8 +54,10 @@
 
 log = logging.getLogger(__name__)
 
-# The primary keys of AnVIL entities stored in TDR do not correspond to the
-# entity IDs used by Azul
+# AnVIL snapshots do not use UUIDs for primary/foreign keys.
```

Comment:
It would be worth mentioning here that each row has a secondary key called datarepo_row_id which IS a UUID and  is used as our internal entity ID.